docs/en/table_design/table_types/aggregate_table.md

```diff
@@ -74,6 +74,319 @@ DISTRIBUTED BY HASH(site_id);
 > - When you create a table, you must specify the bucketing column by using the `DISTRIBUTED BY HASH` clause. For detailed information, see [bucketing](../data_distribution/Data_distribution.md#bucketing).
 > - Since v2.5.7, StarRocks can automatically set the number of buckets (BUCKETS) when you create a table or add a partition. You no longer need to manually set the number of buckets. For detailed information, see [set the number of buckets](../data_distribution/Data_distribution.md#set-the-number-of-buckets).
 
+## Generic aggregate function states
+
+StarRocks supports generic aggregate function states from v3.4.0.
+
+In data analysis and summaries, Aggregate tables reduce the data processed during queries, improving query performance. For large datasets, Aggregate tables are highly effective as they summarize data by dimensions before querying. They also serve as a vital method for incremental aggregate function computation in StarRocks. However, in earlier versions, support is limited to built-in functions such as `SUM`, `MAX`, `MIN`, `REPLACE`, `HLL_UNION`, `PERCENTILE_UNION`, and `BITMAP_UNION`, while in theory, all built-in aggregate functions can be used in aggregate tables. To address this limitation, generic aggregate states is introduced to support the storage of all built-in function states.
```

Comment:
I am confused. In this paragraph it says that all built-in aggregate functions are supported, but on line 103 it says:

Some built-in functions, such as `GROUP_CONCAT`, `WINDOW_FUNNEL`, and `APPROX_TOP_K`, are not supported. For detailed information, see [FunctionSet.java#UNSUPPORTED_AGG_STATE_FUNCTIONS]().

Reply:
@LiShuMing

Reply:
According to RD, the support for the listed functions is still under development, so I clarified this point in the note below.